react-example/src/components/LoginFormWithoutJWT.tsx

```diff
@@ -62,7 +66,7 @@ export const LoginFormWithoutJWT: FC<Props> = ({
 
     const setUser = useEmail ? setEmail : setUserId;
 
-    setUser(user, true)
+    setUser(user)
```

Comment:
I'm still unsure how identityResolution is being used, should we expect it to be passed in?

Reply:
Here, identity resolution _could_ be passed in. The config defaults are set to `mergeOnAnonymousToKnown: true` so just removing the variable here because it's redundant. The values for `IdentityResolution`, i.e., `mergeOnAnonymousToKnown` and `replayOnVisitorToKnown` could be passed in and changed any time `setEmail` or `setUserId` is called.

Reply:
Ah so it's an override and there's logic in the implementation that pulls from the config? Could we label it as something like identityResolutionOverride in the function signatures where it would be an override?

Reply:
Ah so it's an override and there's logic in the implementation that pulls from the config? If so, could we label it as something like identityResolutionOverride in the function signatures where it would be an override?